dask_cuda/local_cuda_cluster.py

```diff
@@ -39,96 +39,95 @@ def __init__(self, *args, **kwargs):
-    If UCX and UCX-Py are also available, it's possible to use InfiniBand and
-    NVLink connections for optimal data transfer performance.
+    Each worker process is automatically assigned the correct CPU cores and network
+    interface cards to maximize performance. If UCX and UCX-Py are available, InfiniBand
+    and NVLink connections can be used to optimize data transfer performance.
 
     Parameters
     ----------
-    CUDA_VISIBLE_DEVICES: str or list
-        String or list ``"0,1,2,3"`` or ``[0, 1, 2, 3]`` to restrict activity to
-        different GPUs.
-    device_memory_limit: int, float or str
-        Specifies the size of the CUDA device LRU cache, which is used to
-        determine when the worker starts spilling to host memory.  This can be
-        a float (fraction of total device memory), an integer (bytes), a string
-        (like ``"5GB"`` or ``"5000M"``), or ``"auto"``, ``0``, or ``None`` to disable
-        spilling to host (i.e., allow full device memory usage). Default is ``0.8``,
-        80% of the worker's total device memory.
-    interface: str
-        The external interface used to connect to the scheduler, usually
-        an ethernet interface is used for connection, and not an InfiniBand
-        interface (if one is available).
-    threads_per_worker: int
-        Number of threads to be used for each CUDA worker process.
-    protocol: str
-        Protocol to use for communication, e.g., ``"tcp"`` or ``"ucx"``.
-    enable_tcp_over_ucx: bool
-        Set environment variables to enable TCP over UCX, even if InfiniBand
-        and NVLink are not supported or disabled.
-    enable_infiniband: bool
-        Set environment variables to enable UCX InfiniBand support, requires
+    CUDA_VISIBLE_DEVICES : str or list of int, optional
+        String (like ``"0,1,2,3"``) or  list (like ``[0, 1, 2, 3]``) to restrict
+        activity to different GPUs.
+    n_workers : int, optional
+        Number of workers. Will override the value of ``CUDA_VISIBLE_DEVICES`` if
+        specified.
+    threads_per_worker : int, default 1
+        Number of threads to be used for each Dask worker process.
+    processes : bool, default True
+        TODO
+    memory_limit : int, float, or str, default "auto"
+        Bytes of memory per process that the worker can use. Can be an integer (bytes),
+        float (fraction of total system memory), string (like ``"5GB"`` or ``"5000M"``),
+        ``"auto"``, or 0 for no memory management.
+    device_memory_limit : int, float or str, default 0.8
+        Size of the CUDA device LRU cache, which is used to determine when the worker
+        starts spilling to host memory. Can be an integer (bytes), float (fraction of
+        total device memory), string (like ``"5GB"`` or ``"5000M"``), ``"auto"``, or 0
+        to disable spilling to host (i.e. allow full device memory usage).
+    data :
+        TODO
+    local_directory : str, optional
+        TODO
```

Comment:
What do these parameters control?

Reply:
`data` is a mapping object where Dask workers store chunks, depending on its internal mechanism it will control device-to-host and host-to-disk spilling, etc. Although it's possible to extend it, it's quite complex to do so and generally users are better off leaving it alone, so I would not document it, and perhaps we could remove it and let it be passed to `**kwargs`, which we could then pop its value so it's not directly exposed in  `LocalCUDACluster`'s API.

`local_directory` is the same as in [Distributed Worker](https://distributed.dask.org/en/latest/worker.html): "Path on local machine to store temporary files".

Reply:
Thanks! I ended up expanding #562 to encompass this as well.